Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -42,7 +42,7 @@
 // On Mac OS X and FreeBSD timeout of -1 causes pcap_open_live to fail so value of 1ms is set here.
 // On Linux and Windows this is not the case so we keep the -1 value
 #if defined(__APPLE__) || defined(__FreeBSD__)
-#define LIBPCAP_OPEN_LIVE_TIMEOUT 1
+#define LIBPCAP_OPEN_LIVE_TIMEOUT 10
```

Comment:
I wonder if it works with a lower value like 2ms 🤔 
Also - maybe separate FreeBSD and Apple? Apple could stay 1ms and we can have a different value for FreeBSD

Reply:
I will do the test, but we need to found the proper fix.
This is just to highlight that it's not an issue with the CI or the user privilege that has changed.

Reply:
maybe we should move to a nonblock version ?

Reply:
Timeout 1ms: 3x KO
Timeout 2ms: 3x KO
Timeout 3ms: 3x KO
Timeout 4ms: 3x KO
Timeout 5ms: 3x KO
Timeout 6ms: 3x KO
Timeout 7ms: 3x KO
Timeout 9ms: 3x KO
Timeout 10ms: 3x OKKK
Timeout 11ms: 3x OKKK
Timeout 15ms: 3x OKKK
Timeout 20ms: 3x OKKK
Timeout 30ms: 3x OKKK

Reply:
@clementperon I'm not sure what this output means 🤔

Reply:
Sorry not clear, I test each configuration 3 times to be sure it was repeatable

Reply:
To be honest I don't know, I have tested that on a FreeBSD VM Aarch64 on MacOS.

I'm not sure we should merge this without understand why no?

Reply:
I have pushed an update with a proper comment

Reply:
> To be honest I don't know, I have tested that on a FreeBSD VM Aarch64 on MacOS.
> 
> I'm not sure we should merge this without understand why no?

Honestly, I'm not sure how we can test it... seems like an internal issue of libpcap on FreeBSD 🤷‍♂️

Reply:
Ideally we would open a bug with  a small sample code to reproduce the issue

Reply:
Sure, you can open a bug. But even if they fix it we need to support older libpcap versions, so we'll need this hack anyway...